### ISSUE TITLE
Blurriness in FMVs when selecting adaptive downsampling (PAL)

### ISSUE THREAD

**Issue by tycooko** [stenzek/duckstation#1368]
I mainly use PAL games, so the following is mostly about them (it might or might not happen to NTSC games, too).

When adaptive downsampling is selected (for appropriate games that use a great mix of 2D backgrounds and 3D characters, like the Final Fantasy and Resident Evil series), what can only be described as spots of blurriness appear to be almost randomly applied to the screen, in various places, when FMVs are shown.
The effect is more and more apparent as the user enhances the resolution, making the image almost 100% blurred out at 16x but also, on the other hand, completely disappearing and leaving the image intact at 1x resolution.

Dino Crisis 2:
<img width="1306" alt="Screen Shot 2021-01-02 at 19 23 02" src="https://user-images.githubusercontent.com/75950926/103464188-060ab280-4d32-11eb-992d-b47acdbcb6b1.png">

Final Fantasy IX:
<img width="1305" alt="Screen Shot 2021-01-02 at 19 28 20" src="https://user-images.githubusercontent.com/75950926/103464194-0c992a00-4d32-11eb-8a60-3f09064661cb.png">

Resident Evil 2:
<img width="1300" alt="Screen Shot 2021-01-02 at 19 37 27" src="https://user-images.githubusercontent.com/75950926/103464198-11f67480-4d32-11eb-897c-c016c552f275.png">

**Comment by Alien-Grey** [comment]
I'm also using Adaptive Downsampling for performance gains and have no iisues with it. It does gives me better performance and looks smooth. I don't have any issues with PAL games..

My settings:
```
[GPU]
Renderer = D3D11
ResolutionScale = 0
Multisamples = 1
UseDebugDevice = false
PerSampleShading = false
UseThread = true
TrueColor = true
ScaledDithering = true
TextureFilter = Nearest
DisableInterlacing = true
ForceNTSCTimings = false
WidescreenHack = false
ChromaSmoothing24Bit = false
PGXPEnable = true
PGXPCulling = true
PGXPTextureCorrection = true
PGXPVertexCache = false
PGXPCPU = false
PGXPPreserveProjFP = false
PGXPTolerance = -1.000000
Adapter = Intel(R) HD Graphics
DownsampleMode = Adaptive


[Display]
CropMode = Overscan
ActiveStartOffset = 0
ActiveEndOffset = 0
LineStartOffset = 0
LineEndOffset = 0
AspectRatio = Auto (Game Native)
Force4_3For24Bit = false
LinearFiltering = false
IntegerScaling = false
PostProcessing = false
ShowOSDMessages = false
ShowFPS = false
ShowVPS = false
ShowSpeed = false
ShowResolution = false
Fullscreen = false
VSync = false
PostProcessChain = 
MaxFPS = 0.000000
UseBlitSwapChain = false
```

**Comment by tycooko** [comment]
@Alien-Grey Hi, thanks for the reply!
Well, that's interesting. Could you give me a couple of games you're trying this on? Or are you trying it with my same games?

**Comment by stenzek** [comment]
It was fixed in https://github.com/stenzek/duckstation/commit/2ec26e54b7f564b5c55931a40c752e5b7aae449b (preview build), forgot to update the issue.

**Comment by tycooko** [comment]
@stenzek Happy to be useful! 
Fortunately, it wasn't one of the "many" Mac-related issues I keep rambling about, like the Cccalibrator issue I've helped you discover and that's now fixed, the memory card corrupting two-blocks saves which has been fixed as well or asking for a custom order of the games, which is totally a Mac thing. Sorry if I'm always giving you issues which are 100% Cupertino's fault. I promise to do better for this new year! ^___^
